packages/interactor/src/definitions/check-box.ts

```diff
@@ -12,9 +12,22 @@ const CheckBoxInteractor = FormField.extend<HTMLInputElement>('check box')
     },
   })
   .actions({
-    check: ({ perform }) => perform((element) => { if(!element.checked) element.click(); }),
-    uncheck: ({ perform }) => perform((element) => { if(element.checked) element.click(); }),
-    toggle: ({ perform }) => perform((element) => { element.click(); }),
+    check: ({ perform }) => perform((element) => {
+      element.indeterminate = false;
+      if(!element.checked) element.click();
```

Comment:
If we are going to set the indeterminate state because that is default browser behavior, then we should we should ensure that the click was not cancelled.

Reply:
Done